packages/node-xmpp-server/lib/S2S/Router.js

```diff
@@ -81,6 +95,7 @@ Router.prototype.acceptConnection = function (socket) {
   var inStream = new IncomingServer({
     streamId: this.generateId(),
     reconnect: false,
+    requestCert: true,
```

Comment:
This line is a symptom of a bug my test exposed. When testing against prosody the code would renegotiate to access the cert and that worked but when I test against itself it hangs w/o this.

I need to figure out what's going on.